src/Web/Models/ErrorViewModel.cs

```diff
@@ -6,6 +6,8 @@ public class ErrorViewModel
     {
         public string RequestId { get; set; }
 
+        public bool IsSuccess { get; set; }
```

Comment:
Shouldn't be IsSuccessful?

Reply:
We can use the shorter version :)))

Reply:
But I will fix it

Reply:
all right, it is fine. Let's go for shorter version